lmnet/lmnet/post_processor.py

```diff
@@ -419,3 +419,74 @@ class Softmax(Processor):
     def __call__(self, outputs, **kwargs):
         results = _softmax(outputs)
         return dict({'outputs': results}, **kwargs)
+
+
+class FormatJoints(Processor):
+
+    def __init__(self, num_dimensions=2, stride=2, confidence_threshold=0.1):
+        """
+        Args:
+            num_dimensions: int, it only supports 2 for now.
+            stride: int, stride = image_height / heatmap_height.
+            confidence_threshold: float, [0, 1], ratio of threshold_value and max_value.
+        """
+        self.num_dimensions = num_dimensions
+        self.stride = stride
+        self.confidence_threshold = confidence_threshold
+
+    def __call__(self, outputs, *args, **kwargs):
```

Comment:
I feel uncomfortable using "outputs" as a variable name for the input argument. Do you have another idea for this variable name?

Reply:
I agree with you. This name is not comfortable.
But since this line of demo code:  https://github.com/blue-oil/blueoil/blob/4928b8ed05023b3fafe1f874c93d2e587c637a6b/output_template/python/lmnet/utils/demo.py#L159
I can not decide this variable name without refactoring.
Current implementation makes we can combine this post processors using `Sequence` in any order. But it does sacrifice the readability. In fact, we may don't need this feature in post process cases..

Reply:
Though I'm not sure, but it looks OK to just remove `outputs=` from the line 159 of `demo.py`. Could you try?

Reply:
It seems ok to remove it. How about changing them in another PR? 
Since all arg of post processors is 'outputs' now.

Reply:
OK, let's leave them as is now.